lib/misc_funcs.sh

```diff
@@ -61,7 +61,7 @@ function export_mix_env() {
     if [ -d $env_path ] && [ -f $env_path/MIX_ENV ]; then
       export MIX_ENV=$(cat $env_path/MIX_ENV)
     else
-      export MIX_ENV=prod
+      export MIX_ENV=${1:-prod}
```

Comment:
This just means use the 1st argument provided to the function, or if there wasn't a first argument, use 'prod'.

That leaves the original usage (`export_mix_env`) unchanged, but you can now provide a fallback other than prod with `export_mix_env 'test'`.

What's nice here is that if a user did specify something like MIX_ENV=dev for a test run (albeit that would be weird), this wouldn't overwrite it.